src/actions/resize/scale.js

```diff
@@ -0,0 +1,24 @@
+import '../../interfaces/IAction';
+
+/**
+ * @implements IAction
+ */
+class Scale {
+  /**
+   * @param {number} scaleWidth
+   * @param {number} scaleHeight
+   */
+  constructor(scaleWidth, scaleHeight) {
+  }
+}
+
+/**
+ * @param {number} scaleWidth
+ * @param {number} scaleHeight
+ * @return {Scale}
+ */
+function scale(scaleWidth, scaleHeight) {
+  return new Scale(scaleWidth, scaleHeight)
```

Comment:
Scale also accepts a single argument, maybe the second should default to something?